### PR TITLE
mgr/status: format byte quantities in base 2 multiples

### PR DESCRIPTION
**Pull request by jcsp**
Fixes: http://tracker.ceph.com/issues/21189
Signed-off-by: John Spray <john.spray@redhat.com>

### CODE REVIEW

**Comment by gmayyyha** [comment]
https://github.com/ceph/ceph/blob/bbf4fc7659dc6df0d409dc53d4dbc92fb58e458f/src/pybind/mgr/dashboard/module.py#L218-L225

@jcsp should also fix this or define as common function?

**Comment by jcsp** [comment]
@gmayyyha the usages of format_dimless in dashboard/module.py are all meant to be base 10 (it's used for rate of requests and rate of events, rather than anything in bytes).

We should get these in a common place at some point but that's a job for another day

File: src/pybind/mgr/status/module.py [hunk 1]
```diff
@@ -157,7 +167,7 @@ def handle_fs_status(self, cmd):
                     activity = ""
 
                     if state == "active":
-                        activity = "Reqs: " + self.format_dimless(
+                        activity = "Reqs: " + self.format_units(
```
Comment (tchaikov) [inline src/pybind/mgr/status/module.py]: should use `format_dimless()`, i think.

Comment (jcsp) [reply src/pybind/mgr/status/module.py]: you're right, fixed.

**Comment by jcsp** [comment]
@tchaikov quick re-review?

**Pull request merged**